build/excludefiles.txt

```diff
@@ -6,9 +6,21 @@
 .travis.yml
 *.git*
 *.idea*
+app/console
 app/phpunit.xml.dist
+bin/codecept
```

Comment:
not sure about this
What about to prevent access with .htaccess and keep this folder content?

Reply:
Please use .htaccess only

Reply:
Fixed in e1a9e8909db0be518b27befc1fb87e0bad724f65